### PR TITLE
Added versioning information 

### PR DESCRIPTION
**Pull request by aburdenthehand**
Based on information https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md

First contribution so hopefully this works :) (and CLA is working)

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6442)

### CODE REVIEW

**Comment by aburdenthehand** [comment]
Should add this is for #6098

**Comment by aburdenthehand** [comment]
Not sure why it's failing CLA check. That should be sorted...

File: docs/home/index.md [hunk 1]
```diff
@@ -34,3 +34,21 @@ The [Tools](/docs/tools/) page contains a list of native and third-party tools f
...
+Kubernetes has a _X.Y.Z_ versioning scheme, where _X_ is the major version, _Y_ is the minor version, and _Z_ is the patch version. 
+
+Kubernetes is supported for three minor versions at a time. This includes the current release version and two previous versions. 
+
+See the [Kubernetes Release](https://github.com/kubernetes/kubernetes/releases) page on GitHub for the latest release information.
+
+### Minor Versions
+
+Different components are expected to be compatible across different version skew relative to the master. Nodes may lag master by up to two versions, but not exceed the master version. Clients may lag master by one version and may exceed master up to one version.
+
```
Comment (steveperry-53) [inline docs/home/index.md]: I stumbled over the first sentence under Minor Versions. I suggest changing it to something like this:
A certain amount of skew is permissible between master components, node components, and the kubectl client.

Comment (aburdenthehand) [reply docs/home/index.md]: Thanks for the review @steveperry-53 !
Updated almost as suggested ("A certain amount of version skew...")

**Pull request merged**